assets/js/src/common/listings/newsletter_stats.tsx

```diff
@@ -25,23 +25,17 @@ const NewsletterStats = ({
   const openedDisplay = MailPoet.Num.toLocaleFixed(opened, 1);
   const clickedDisplay = MailPoet.Num.toLocaleFixed(clicked, 1);
 
-  let openedAndClickedStats = (
+  let clickedAndOpenedStats = (
     <div className="mailpoet-listing-stats-opened-clicked">
       <div className="mailpoet-listing-stats-percentages">
-        {openedDisplay}
+        {clickedDisplay}
         %
         <br />
-        {clickedDisplay}
+        <span className="mailpoet-listing-stats-percentages-opens">{openedDisplay}</span>
         %
```

Comment:
I believe we should put this percentage symbol inside span.mailpoet-listing-stats-percentages-opens so that it is displayed without bold as well?